make/tests

```diff
@@ -38,16 +38,28 @@ stampy "${METRICS}" "$0" "make-tests::${POD_NAME}::setup" end
 
 stampy "${METRICS}" "$0" "make-tests::${POD_NAME}::create" start
 
+# Replace the fixed secret in the relevant task definition with the
+# actual name as pulled from the cluster under test.
+
+capsecret="$(kubectl get pod api-0 --namespace "${NAMESPACE}" -o jsonpath='{@.spec.containers[0].env[?(@.name=="MONIT_PASSWORD")].valueFrom.secretKeyRef.name}')"
+#echo cap-secret = $capsecret
+
+remove_test_config () { rm test.yaml ; }
```

Comment:
Should that use `mktemp`? If the developer happens to be playing with things and has a temporary file named `test.yaml` it'll get deleted...

Reply:
Yes, will fix.

Reply:
Done with commit 619e92637782a7acd2566211ff3fec2cb0f238b3